CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # Unreleased
-
+- Added to `WindowAttributes` the attributes `center_window: bool` and `start_monitor: i16`. Window centering is by default enabled.
```

Comment:
`start_monitor` is `Option<MonitorId>`, not `i16`. Window centering should default to being disabled.